__tests__/workers/postUpdated.ts

```diff
@@ -237,6 +237,47 @@ it('should update post and modify keywords', async () => {
   expect(postKeywords.length).toEqual(2);
 });
 
+it('should save keywords in cleaned versions', async () => {
```

Comment:
maybe also `should clean X and Y` just so we better understand how are keywords supposed to be cleaned down the line.